sys/include/trickle.h

```diff
@@ -42,19 +42,22 @@ typedef struct {
 typedef struct {
     uint8_t k;                      /**< redundancy constant */
     uint32_t Imin;                  /**< minimum interval size */
-    uint8_t Imax;                   /**< maximum interval size, described as a number of doublings */
+    uint8_t Imax;                   /**< maximum interval size, described as a number of
```

Comment:
Could you move up Imax? two consequtive uint8_t will take 4 byte in the struct, like this they will take up 8.